pkgdiff

```diff
@@ -3,6 +3,12 @@
 
 set -e -x
 
+build_system=0
+if [[ $1 == --build-system ]]; then
+	build_system=1
+	shift
+fi
```

Comment:
I'm sorry but that's not consistent with my coding style. I think the following would be more future-proof:

```suggestion
mode=
while [[ ${#} -gt 0 ]]; do
	case ${1} in
		--build-system)
			mode=build-system
			;;
		*)
			break
			;;
	esac
	shift
done
```